d3rlpy/cli.py

```diff
@@ -355,27 +355,21 @@ def play(
 @cli.command(short_help="Install additional packages.")
 @click.argument("name")
 def install(name: str) -> None:
-    if name == "atari":
-        subprocess.run(
-            ["pip3", "install", "-U", "gym[atari,accept-rom-license]"],
-            check=True,
-        )
-    elif name == "d4rl_atari":
-        subprocess.run(["d3rlpy", "install", "atari"], check=True)
-        subprocess.run(
-            ["pip3", "install", "git+https://github.com/takuseno/d4rl-atari"],
-            check=True,
-        )
-    elif name == "d4rl":
-        subprocess.run(
-            [
-                "pip3",
-                "install",
-                "git+https://github.com/Farama-Foundation/D4RL",
-            ],
-            check=True,
-        )
-        subprocess.run(["pip3", "install", "-U", "gym"], check=True)
-        subprocess.run(["pip3", "uninstall", "-y", "pybullet"], check=True)
-    else:
-        raise ValueError(f"Unsupported command: {name}")
+    match name:
```

Comment:
Can you revert this change to use `if-else` conditions? Because `match` is supported since Python3.10, however `d3rlpy` supports Python3.8 or above. It's a little too early to use this Python feature.

Reply:
yeah figured you wouldnt want this as its gonna break all <3.10 but thought it was worth a try.  its a shame it will never really be a used feature as i tend to find it pretty apt for exactly this situation